### ISSUE TITLE
vendor/tmux-mem-cpu-load should follow branch

### ISSUE THREAD

**Issue by zembrowski** [tony/tmux-config#13]
Hi Tony,

could you set the tmux-mem-cpu-load submodule to follow the master branch of https://github.com/thewtex/tmux-mem-cpu-load?

Alternatively you could change one line in the README:
> git submodule update **--remote**

You see the args changed in the tmux.conf for tmux-mem-cpu-load with commit https://github.com/tony/tmux-config/commit/0d6943ea186c9c5cb4f40431923b079a25105e60, but the submodule follows an old version. Therefore, on a fresh install tmux-mem-cpu-load doesn't seem to work. Might be frustrating for a new user.

Thank you in advance.

Best regards,
Krzysztof

**Comment by tony** [comment]
Hey @zembrowski! That sounds like a plan!, would you like to submit a pull request for it?

**Comment by tony** [comment]
I apologize for the late response.

I updated the version of tmux-mem-cpu-load to v3.2.2.

If we switch to the submodule to follow master, the instructions the README provides could fall out of sync. The command inside .tmux.conf would also need to be updated manually by the user.

We could add instructions on how to use the latest tmux-mem-cpu-load and refer them to tmux-mem-cpu-load's documentation.

What do you think?

**Comment by zembrowski** [comment]
Hi Tony,

Thanks for the response.

The issue I described back in April was that on a fresh install tmux-mem-cpu-load didn't work out-of-the-box following the README instruction.

Tried it now on a new install and it works like a charm, so problem solved.

And yes, you are right - there should be an instruction on how to use the lastest tmux-mem-cpu-load module. Additionally it would be very good to update the submodule on a regular basis and check if any adjustments in the .tmux.conf are necessary.

Best regards,
Krzysztof
